code/game/objects/items/storage/pouch.dm

```diff
@@ -335,7 +335,7 @@
 		if(M.contents.len)
 			if(contents.len < storage_slots)
 				to_chat(user, "<span class='notice'>You start refilling [src] with [M].</span>")
-				if(!do_after(user, 15, TRUE, 5, BUSY_ICON_GENERIC))
```

Comment:
Is the removal of the icon intended here?

Reply:
same.